### ISSUE TITLE
error looking for non-existent file

### ISSUE THREAD

**Issue by wcbeard** [lukexi/stack-ide-sublime#13]
When I restart Sublime to my hello world project, I get an error message "can't find file: /Users/me/first-project/Lib". Debug-level ST3 console and `stack ide start first-project` output are [here](https://gist.github.com/d10genes/26d52bb80a0521f4c700) (starting from command line doesn't seem to have same problem).

I'm following the directions [here](https://github.com/commercialhaskell/stack/wiki/New-to-Haskell) for building the hello world project. This creates a directory with the following structure

```
➜  first-project  tree
.
├── LICENSE
├── Setup.hs
├── app
│   └── Main.hs
├── first-project.cabal
├── src
│   └── Lib.hs
├── stack.yaml
└── test
    └── Spec.hs
```

The `library` section of `first-project.cabal` looks like

```
library
  hs-source-dirs:      src
  exposed-modules:     Lib
  build-depends:       base >= 4.7 && < 5
                     , bytestring
                     , http-conduit
  default-language:    Haskell2010
```

so I'm not sure why it's looking for `/Users/me/first-project/Lib`.

```
➜  ~  stack --version
Version 0.1.3.1
```

**Comment by jcpetruzza** [comment]
This looks strange and I wonder if it is related to #12. Anyway, could you try running this time the following?

```
echo "{'tag': 'RequestGetSourceErrors', 'contents': []}" | stack ide start first-project
```

If that way you get the error, could you then repeat it passing the `--verbose` flag to `stack` and paste that log?

**Comment by wcbeard** [comment]
Not sure what it means, but [here is the output](https://gist.github.com/d10genes/26d52bb80a0521f4c700#file-cli2). It has the word 'error', but I think it's  different.

Edit: [here](https://gist.github.com/d10genes/26d52bb80a0521f4c700#file-cli2-verbose) is the verbose flag

**Comment by jcpetruzza** [comment]
Right, that was my fault, sorry! Instead of that, please run `stack ide start first-project --verbose` and paste the following through stdin: `{'tag': 'RequestGetSourceErrors', 'contents': []}`

**Comment by wcbeard** [comment]
Is this what you mean? https://gist.github.com/d10genes/26d52bb80a0521f4c700#file-verbose-2

**Comment by jcpetruzza** [comment]
Argh, wrong type of quotes, sorry (sigh). It should have been: `{"tag": "RequestGetSourceErrors", "contents": []}`

**Comment by wcbeard** [comment]
Aha! With double quotes, I get `{"tag":"ResponseGetSourceErrors","contents":[{"errorSpan":{"tag":"TextSpan","contents":"<no location info>"},"errorKind":"KindError","errorMsg":"can't find file: /Users/chrisbeard/first-project/Lib"}]}`. Does this mean the problem is in the domain of the stack-ide people?

**Comment by jcpetruzza** [comment]
Yes, it looks like the problem is on their end. Could you open an issue there?

**Comment by jcpetruzza** [comment]
More people with this problem: https://github.com/commercialhaskell/stack-ide/issues/56

**Comment by tomv564** [comment]
I believe this problem may have been fixed (using stack 1.5 and stack-ide fd8c3629), but updating to latest stack-ide will result in new errors. I suggest we close this issue.

**Comment by lukexi** [comment]
Should hopefully be good now with latest stack + stack-ide + stack-ide-sublime, please reopen if not!!
